CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
   * Updated the index hosted at the root prefix to point to the updated routes.
   * Legacy routes hardcoded with the `/api/prom` prefix now respect the `-http.prefix` flag.
 * [CHANGE] The metrics `cortex_distributor_ingester_appends_total` and `distributor_ingester_append_failures_total` now includes a `type` label to differentiate between `samples` and `metadata`. #2336
+* [CHANGE] The metrics for number of chunks and bytes flushed to the chunk store are renamed from: #2463
+  * `cortex_ingester_chunks_stored_total` > `cortex_chunk_store_stored_chunks_total`
```

Comment:
Would it make sense to keep original names, and name new metrics with `cortex_querier_` prefix, instead of using `cortex_chunk_store` prefix?

Reply:
We'll be using the deletes package to track this and soon the backfilling / migration tool to track costs as well. So ingester is not the only thing that will read and write chunks.

Reply:
It would be nice to know which component is fetching how many chunks in that case, a combined metric sounds less useful. Maybe an additional label?

Reply:
It's currently not possible to track which component is doing the request without significant plumbing. And given our deletes and migrations are one offs, I'd rather go down that route just yet.